resources/zib/zib-VisualAcuity.xml

```diff
@@ -0,0 +1,301 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-VisualAcuity" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-VisualAcuity" />
+  <name value="ZibVisualAcuity" />
+  <title value="zib VisualAcuity" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="Visual acuity is a measurement of sharpness of vision. It is a measure of the smallest details that someone can still distinguish." />
+  <purpose value="This Observation resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) VisualAcuity-v1.0(2020EN)](https://zibs.nl/wiki/VisualAcuity-v1.0(2020EN)). Because of some irregularities that were found in the 2020 version of the zib, concept names, descriptions and DefinitionCodes were taken from version [v2.0(2021EN)](https://zibs.nl/wiki/Visus-v2.0(2021NL))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-visualacuity-v1.0-2020EN" />
+    <uri value="https://zibs.nl/wiki/VisualAcuity-v1.0(2020EN)" />
+    <name value="zib VisualAcuity-v1.0(2020EN)" />
+  </mapping>

```

Comment:
Should we also add (or replace?) the mapping to the 2021 version, or is the text added to the purpose sufficient?

Reply:
There are addenda for all the changes in the profile, so there's no need to add the mapping and in fact there's no need for the text in `.purpose`. I'll remove that text.

Reply:
I removed all 2021 preadopts, as this is not in line with the profiling guidelines.